components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/configuration/HumanTaskServerConfiguration.java

```diff
@@ -71,6 +71,9 @@
 
     private boolean uiRenderingEnabled = false;
 
```

Comment:
Renamed variables in commit https://github.com/wso2/carbon-business-process/commit/c037c2607734679028dc96c59761857176b6dfdb